Code/GraphMol/SynthonSpaceSearch/SynthonSpace.h

```diff
@@ -78,6 +78,8 @@ struct RDKIT_SYNTHONSPACESEARCH_EXPORT SynthonSpaceSearchParams {
              // times, a lower number will give faster searches at the
              // risk of missing some hits.  The value you use should have
              // a positive correlation with your FOMO.
+  long long int timeOut{
```

Comment:
Any reason not to use

```
std::int64_t
```

Reply:
I guess I hovered over the relevant value in CLion and it told me it was a long long int.

Reply:
I think std::chrono::duration::count returns mostly "long long int".  I prefer int64_t but YMMV.

Reply:
> Any reason not to use
> 
> ```
> std::int64_t
> ```

I would prefer this too. I like the types that are specific about their size.